### PR TITLE
Add swoop/swiper and highlight to Symbol Highlight Transient State

### PR DESCRIPTION
**Pull request by adrsta**

### CODE REVIEW

**Comment by nixmaniack** [comment]
Please don't forget `ivy` users! 😄

**Comment by adrsta** [comment]
something like this?
somehow swiper doesn't work for me (neither in a fresh config)

**Comment by nixmaniack** [comment]
@adrsta swiper seems [broken](https://github.com/abo-abo/swiper/issues/683) at the moment - neither could I use it after recent upgrade.

**Comment by adrsta** [comment]
Like so?

**Comment by adrsta** [comment]
I also amended a permanent [h]ightlight binding now

**Comment by adrsta** [comment]
@TheBB @d12frosted could you approve the changes?

**Comment by d12frosted** [comment]
@adrsta sorry, I missed your update.

File: layers/+spacemacs/spacemacs-editing-visual/packages.el [hunk 1]
```diff
@@ -263,11 +267,15 @@
         ("d" ahs-forward-definition)
         ("D" ahs-backward-definition)
         ("e" ahs-to-iedit :exit t)
+        ("h" highlight-symbol-at-point)
         ("n" spacemacs/quick-ahs-forward)
         ("N" spacemacs/quick-ahs-backward)
         ("p" spacemacs/quick-ahs-backward)
         ("R" ahs-back-to-start)
         ("r" ahs-change-range)
+        ("s" (if (configuration-layer/layer-usedp 'ivy)
```
Comment (TheBB) [inline layers/+spacemacs/spacemacs-editing-visual/packages.el]: Usually you'd modify `spacemacs-symbol-highlight-transient-state-add-bindings` in both the ivy and helm layers, as done [here](https://github.com/syl20bnr/spacemacs/blob/develop/layers/+completion/helm/packages.el#L49-L52) and [here](https://github.com/syl20bnr/spacemacs/blob/develop/layers/+completion/ivy/packages.el#L38-L41) for the other bindings. I'm not sure why you chose not to do this but I recommend that you do.

Comment (adrsta) [inline layers/+spacemacs/spacemacs-editing-visual/packages.el]: It's now changed according to recommendation @TheBB

**Comment by syl20bnr** [comment]
Thank you ! 👍 
I removed the permanent highlight though, see explanation in commit https://github.com/syl20bnr/spacemacs/commit/d0401219
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**